01matrix.java

```diff
@@ -0,0 +1,58 @@
+// Time Complexity : O(nm)
+// Space Complexity : O(1)
```

Comment:
SC shouldn't be O(1) for this solution. I can see that you're using a Queue.

Reply:
I changed the space complexity.